src/ui/WcPanel.tsx

```diff
@@ -387,10 +430,139 @@ const WcPanel = ({ records, attributes, query, actions, adapter, creatingAdapter
 
   const saveAdapterCode = function() {
     chrome.storage.local.set({ [_adapterKey]: adapterCode }, function() {
-console.log("saved changes");
+    console.log("saved changes");
     });
   }
 
+  // This pattern matches every thing after and including any one of these symbols: = ( + - * /
+  const regex = /[=(\+\-\*\/][^=(\+\-\*\/]*$/; 
+    
+  // Teach Autosuggest how to calculate suggestions for any given input value.
+  const mathSymbols = {"Plus": "+", "Minus": "-", "Multiply": "*", "Divide": "/"};
+  const attributeNames = attributes.map(attribute => attribute.name);
+  const mathSuggestions = Object.keys(functions)
+    .filter(functionName => Object.keys(mathSymbols).indexOf(functionName) != -1)
+    .reduce((obj, functionName) => {
+      const symbol = mathSymbols[functionName];
+      obj[symbol] = functions[functionName];
+      return obj; 
+    }, {});
+  const allSuggestionNames = Object.keys(functions)
+    .sort()
+    .filter(functionName => ["Plus", "Minus", "Multiply", "Divide"].indexOf(functionName) == -1)
```

Comment:
Can this just be `functionName => Object.keys(mathSymbols).indexOf(functionName) === -1` ?